README.md

```diff
@@ -1,2 +1,4 @@
 # COSC_Hacktoberfest_PiratesOnClouds_PMS
 Project Management System for managing Projects in the Industry by Team Pirates On Clouds at Hacktoberfest by COSC at CBIT on October 13, 2018
+
+1.[K Girish Chandra](https://github.com/gitgirish2/)
```

Comment:
Please add the heading

## Contributors

Keep the commit messages and PR name logical